src/main/java/org/cascadebot/cascadebot/commands/informational/ServerInfoCommand.java

```diff
@@ -25,7 +25,12 @@ public void onCommand(Member sender, CommandContext context) {
         Guild guildForInfo = context.getGuild();
 
         if (context.getArgs().length > 0) {
-            guildForInfo = CascadeBot.INS.getShardManager().getGuildById(context.getArg(0));
+            try {
+                guildForInfo = CascadeBot.INS.getShardManager().getGuildById(context.getArg(0));
+            } catch (NumberFormatException e) {
+                context.getUIMessaging().replyUsage();
```

Comment:
Why are you replying usage? you should say they gave an invalid guild id.

Reply:
"We cannot find that guild" or are we going to be specific here on that its guild ID

Reply:
You should say something along the lines of they need to use a guild id, as otherwise people will enter a guilds name and then get confused

Reply:
"We could not find that guild ID!" enough?

Reply:
I was thinking more along the lines of "That isn't a valid guild id", or "%s isn't a valid guild id" (obviously replacing %s with that the user entered). This isn't the guild not being able to be found this is them entering the wrong information.